### PR TITLE
PostgreSQL vs MySQL

### LINKED ISSUE #1980: Postgresql vs Mysql
### Proposed title of article

Postgresql vs Mysql

### Introduction paragraph:

We will focus on the special features that have made Postgresql and Mysql remain relevant and widely used database management system. The two were developed slightly over two decades ago and have ranked among the best and most popular database systems. So exactly why have they stood the test of time? Lets find out!

### Key takeaways: 

- What is Postgresql
- What is Mysql
- The key features of both
- Conclusion.

### References: 
N/A

### Templates to use as guides
N/A

### PR DESCRIPTION
**Pull request by amchilos01**
this fixes #1980

### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by lalith1403** [comment]
@zolomohan, I see you have plenty number of PRs assigned. I would recommend you wait till all your PRs are published before taking on new PRs. I will be reviewing this PR. Thank you :)

cc: @hectorkambow

**Comment by zolomohan** [comment]
@lalith1403 Sure go ahead.

**Comment by lalith1403** [comment]
@amchilos01 I am having trouble editing the files submitted. Can you check for edit access rights? This isn't the case with all PRs. Probably something to do with this PR. 

cc: @hectorkambow

**Comment by amchilos01** [comment]
@lalith1403 I have allowed edits. Kindly let me know if you are now able to edit. apologies for the oversight.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by amchilos01** [comment]
Hey @lalith1403, did you manage to now access this PR?

**Comment by lalith1403** [comment]
> Hey @lalith1403, did you manage to now access this PR?

Yes. Thank you.

**Comment by lalith1403** [comment]
@amchilos01 Check the author folder. We will need a headshot image and a short bio to include on your profile page. I am supposing you are submitting the article for the first time since it says first-time contributor.

File: content/articles/postgresql-vs-mysql/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Managing data for operations with heavy analysis is more accessible with PostgreSQL. It is an object-relational database, hence includes features like function overloading where one feature can do multiple tasks and inherit tables where properties of one table can be 'inherited' in another table. It also supports materialized views so that users can save a Structured Query Language (SQL) query on analysis on a disc as a physical table. This increases the order of magnitude speeds of complex SQL queries. As a result, the queries can interact more effectively with other SQL queries.
+
+The performance of PostgreSQL was also improved by its index design, which facilitates a comparatively quicker execution of CRUD (Create Read Update Delete) activities.
```
Comment (lalith1403) [inline content/articles/postgresql-vs-mysql/index.md:40]: Can you talk more about index design? A short overview of what it offers and why it was introduced?

File: content/articles/postgresql-vs-mysql/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+The database supports [InnoDB](https://dev.mysql.com/doc/refman/5.6/en/innodb-introduction.html) formats, which facilitates transactions in the system. The InnoDB formats also allow the creation of InnoDB tables. MySQL supports foreign key constraints for these tables hence eliminating unwanted cross-references in linked tables.
+
+The database is also equipped with JSON data validation, which rejects invalid JSON data from being entered into your projects.
+
+MySQL facilitates an easier transition to a new password in codes by supporting dual passwords for the entered data. A user does not need to edit an entire application when updating the password.
+
+Other features of MySQL include support for multi-version concurrency control, multi-threads using Kernel Threads, and huge data capability. The ANSI SQL standard database can handle rows exceeding 50 million.
+
+![ProstreSQL Vs. MySQL](/engineering-education/postgresql-vs-mysql.png)
+
```
Comment (lalith1403) [inline content/articles/postgresql-vs-mysql/index.md:69]: A table summarizing the two databases on the basis of features discussed in the article would help the reader gain more insights at the end of the article.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
A few minor changes and we should be able to proceed with the final review.

**Comment by amchilos01** [comment]
@lalith1403 the recommended changes have been successfully committed.

**Comment by lalith1403** [comment]
I see you have repeated yourself in multiple instances. This repetition reduces the quality of the article. I strongly suggest you remove these repetitions and add more information if necessary. An example would be the sentence related to GDPR and EU laws. You use it in 2 sections, and the sentences are the same.

**Comment by ahmadmardeni1** [comment]
This PR has the old structure @hectorkambow

**Comment by ahmadmardeni1** [comment]
Please delete these two files: `.vs/engineering-education/v16/.suo` &  `.vs/slnx.sqlite` and why are they in your PR? 
Also, change the dimension of your personal-picture to 600X400 @amchilos01

**Comment by louisefindlay23** [comment]
Have deleted the .vs code files, @ahmadmardeni1.

**Comment by ahmadmardeni1** [comment]
Good work gentlemen @amchilos01 @lalith1403 
And congrats on your first article.
Please go through my changes.

Please make sure to check the author folder in the next reviews.

**Pull request merged**